share/nb.po

```diff
@@ -2,12 +2,73 @@ msgid ""
 msgstr ""
 "Project-Id-Version: 1.0.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-10-14 11:25+0200\n"
-"PO-Revision-Date: 2021-10-14 11:25+0200\n"
-"Last-Translator: mats.dufberg@iis.se\n"
+"POT-Creation-Date: 2021-12-08 15:14+0100\n"
+"PO-Revision-Date: 2021-12-08 15:36+0100\n"
+"Last-Translator: richard.persson@norid.no\n"
 "Language-Team: Zonemaster project\n"
 "Language: nb\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
+msgid "Domain name required"
+msgstr "Domenenavn påkrevd"
+
+msgid ""
+"The domain name is not a valid IDNA string and cannot be converted to an A-"
+"label"
+msgstr ""
+"Domenenavnet er ikke en gyldig IDNA streng og kan ikke konverteres til en "
+"A-post"
```

Comment:
Is "A-post" really correct here? Below you have used "label" for "label"? In the translation you have "post" for "record".

Reply:
My bad, embarrassed.